pepper-apis/src/main/java/org/broadinstitute/ddp/model/study/StudyLanguage.java

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.ddp.model.study;
+
+public class StudyLanguage {
+    private String languageCode;
+    private String displayName;
+    private boolean isDefault;
+    private long studyId;
+    private long languageId;
```

Comment:
We typically don't return these database PK IDs in our JSON responses or expose in anyway via our APIs. Can you mark them as `transient` so they don't show up in JSON?
```java
private transient long studyId;
private transient long languageId;
```